### PR TITLE
2890 add assigned patients to progress tab

### PR DESCRIPTION
**Pull request by kpethtel**
**Story card:** [ch2890](https://app.clubhouse.io/simpledotorg/story/2890/add-total-assigned-patients-to-progress-bar-web-view)

## Because

We want to display the total assigned patients to the user in the android app. 

## Test instructions

This can be tested either with the android app (more difficult) or by changing headers (see instructions in the readme) and going to: http://localhost:3000/api/v3/analytics/user_analytics.html
Confirm that web view contains the new assigned patients section and matches the mocks.

### CODE REVIEW

**Review by harimohanraj89 — CHANGES_REQUESTED** [review]
Some suggestions from me!

File: app/views/api/v3/analytics/user_analytics/show.html.erb [hunk 1]
```diff
@@ -75,6 +75,13 @@
     </div>
   </div>
 
+  <!-- display number of patients assigned to facility -->
+  <div class="card assigned-patients-card">
+    <p class="assigned-count"><%= @user_analytics.current_facility.assigned_hypertension_patients.count %></p>
```
[the remark this answers is not in the document]
Comment (kpethtel) [reply app/views/api/v3/analytics/user_analytics/show.html.erb:80]: Will do.

File: app/views/api/v3/analytics/user_analytics/show.html.erb [hunk 2]
```diff
@@ -75,6 +75,13 @@
...
+    <p class="assigned-label"> <%= t("analytics.assigned_patients_label") %></p>
```
[the remark this answers is not in the document]
Comment (kpethtel) [reply app/views/api/v3/analytics/user_analytics/show.html.erb:81]: Will do.

**Review by rsanheim — APPROVED** [review]
LGTM

**Review by harimohanraj89 — APPROVED** [review]
:shipit:

**Comment by kpethtel** [comment]
> :shipit:

Actually, this got put on ice for now. What should I do with this @claudiovallejo ?

**Comment by claudiovallejo** [comment]
> > :shipit:
> 
> 
> 
> Actually, this got put on ice for now. What should I do with this @claudiovallejo ?

Updated designs for the progress tab won't be ready for another 1 - 2 weeks. Not sure if it's best to close this PR for now and reopen later @kpethtel?

**Pull request closed without merging**